docs/advanced_topics/api/v2/usage.rst

```diff
@@ -478,6 +478,9 @@ This can also be used across relationships. For example,
 ``?fields=body,feed_image(width,height)`` will nest the ``width`` and ``height``
 of the image in the response.
 
+Notice if you use fields in combination with ``child_of`` you also need the ``type``
+parameter to use fields.
+
```

Comment:
Just reading this, it might be best to split this change out into its own PR, will be a small one and can be reviewed in isolation.